### PR TITLE
Add section on yt_idv

### LINKED ISSUE #164: Describe yt_idv
A brief couple sentences about [yt-idv](https://github.com/yt-project/yt_idv/) would go well in `35.viz_and_volrendering.md`.

### PR DESCRIPTION
**Pull request by matthewturk**
Closes #164

### CODE REVIEW

**Comment by matthewturk** [comment]
@chrishavlin do you think you could take a look?

**Comment by chrishavlin** [comment]
> @chrishavlin do you think you could take a look?

Sure! Will do!

File: content/35.viz_and_volrendering.md [hunk 1]
```diff
@@ -168,4 +168,23 @@ This particular set of transfer functions, with what amounts to multi-channel, m
...
+`yt_idv` is built on `PyOpenGL` and provides support for grid-based, particle-based and finite element datasets, with a heavy emphasis on the grid-based datasets.
+While the process of volume rendering is interesting, with many different fascinating areas of inquiry and opportunities for optimization, `yt_idv` is notable for its architecture more than its algorithms and optimization.
+
+`yt_idv` is built using the Traitlets library, and utilizes the immediate-mode graphical user interface system "Dear ImGUI" for presentign a user interface.
+This allows for the system to be largely data-driven; frame redraws are only executed when a parameter changes (such as the camera path, transfer function, etc) and new parameters for the visualization can be easily exposed to the user interface.
+Furthermore, this allows shaders to be dynamically recompiled only as necessary.
+
+In addition to this, the shaders themselves are specified in a configuration file, and they are designed for interoperability.
```
Comment (chrishavlin) [inline content/35.viz_and_volrendering.md:185]: Would be good to clarify that the shader code itself isn't in the yaml, e.g., : 

```suggestion
In addition to this, the shaders themselves are specified in a configuration file that links GLSL source files into multi-component shaders, allowing declarative construction of shader pipelines and improving interoperability.
```

**Review by chrishavlin — APPROVED** [review]
Looks great! One typo to fix and one suggestion.

**Pull request merged**